components/UploadForm.vue

```diff
@@ -121,6 +121,25 @@
         </Dialog>
       </Card>
     </div>
+    <div
+      class="
+        min-w-[648px]
+        flex
+        items-center
+        justify-between
+        px-[43px]
```

Comment:
Should be design mistake ☺️
```suggestion
        px-[44px]
```